<?xml version="1.0"?>
<launch>
  <!-- Start the unified node -->
  <node name="swiftpro_unified_node" pkg="swiftpro" type="swiftpro_unified_node" output="screen">
  </node>

  <!-- Start RVIZ visualization node (if needed) -->
  <!-- <node name="swiftpro_rviz_node" pkg="swiftpro" type="swiftpro_rviz_node" output="screen">
  </node> -->
</launch> 